csrc/mmdeploy/net/rknn/rknn_net.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) OpenMMLab. All rights reserved.
+
+#ifndef MMDEPLOY_SRC_NET_RKNN_RKNN_NET_H_
+#define MMDEPLOY_SRC_NET_RKNN_RKNN_NET_H_
+
+#include "mmdeploy/core/mpl/span.h"
+#include "mmdeploy/core/net.h"
+#include "rknn_api.h"
+
+namespace mmdeploy {
```

Comment:
namespace mmdeploy -> namespace mmdeploy::framework.

namespace mmdeploy is used for C++ API

For inner C++ classes, we use `namespace mmdeploy::framework`. Please refer to #938

Reply:
Done